packages/nc-gui/components/cell/attachment/index.vue

```diff
@@ -209,15 +210,29 @@ const rowHeight = inject(RowHeightInj, ref(1.8))
               <div class="text-center w-full">{{ item.title }}</div>
             </template>
             <div v-if="isImage(item.title, item.mimetype ?? item.type)">
-              <div class="nc-attachment flex items-center justify-center" :class="{'ml-2':active}" @click.stop="selectedImage = item">
+              <div
+                class="nc-attachment flex items-center justify-center"
+                :class="{ 'ml-2': active }"
+                @click.stop="selectedImage = item"
+              >
                 <LazyCellAttachmentImage
-                  class="max-h-[1.8rem] max-w-[1.8rem]"
+                  class="h-[1.8rem] w-[1.8rem]"
                   :alt="item.title || `#${i}`"
+                  :class="{
+                    'h-[2.8rem] w-[3.8rem]': rowHeight === 2,
+                    'h-[4.8rem] w-[5.8rem]': rowHeight === 4,
+                    'h-[5.8rem] w-[6rem]': rowHeight === 6 || isDrawerExist(),
```

Comment:
I am assuming u are using `isDrawerExist` to know ExpandedForm is open. Better way would be to use 
`IsExpandedFormOpenInj`